screens/OrderFormII.js

```diff
@@ -73,6 +76,32 @@ class OrderFormII extends React.Component {
     )
   }
 
+  addPinnedObjtoOrderItems = productId => {
```

Comment:
I believe adding pinned product to order should follow the same flow as other products (i.e. going in into quantity selection screen)